composer.json

```diff
@@ -33,6 +33,7 @@
     "ext-zip": "Enables the use of ZipArchive to export AMP Stories."
   },
   "config": {
+    "optimize-autoloader": true,
```

Comment:
This ensures that the classmap for `includes/amp/` is regenerated with `composer install`. Otherwise, I think it's necessary to run `composer install -o`.

Reply:
This should be removed, and a corresponding change should be made to the build process instead.

If an optimized autoloader is forced, then you'll have to constantly rebuild the plugin on filesystem changes even during development.

Reply:
Sure, it's removed in https://github.com/ampproject/amp-wp/pull/3659/commits/4ac1d2528732709d8cf3daf9152908ab91e7f20d. Next, I'll apply in in the build processes.

Reply:
https://github.com/ampproject/amp-wp/pull/3659/commits/c7d5d6e69f94289144d67b9a6adae85f8c7fa080 adds the `-o` flag to `composer install` only for production build processes and when creating a pre-release.

Maybe it should should also be added to `.travis.yml`, but I thought the optimization [wouldn't be worth](https://getcomposer.org/doc/articles/autoloader-optimization.md#autoloader-optimization) the trouble in debugging if the build is cached:

>The performance gains are not worth the trouble in a development setting.